cpplint/cpplint.py

```diff
@@ -5847,7 +5846,7 @@ def CheckRedundantOverrideOrFinal(filename, clean_lines, linenum, error):
            'already declared as "final"'))
 
 _RE_PATTERN_GFLAG_REGISTER_FLAG_VALIDATOR_API = re.compile(r'RegisterFlagValidator')
-_RE_PATTERN_INCLUDE_GFLAG_API = re.compile(r'#include\s+<gflags/gflags.h>')
+_RE_PATTERN_INCLUDE_GFLAG_API = re.compile(r'^\s*#include\s+<gflags/gflags.h>')
```

Comment:
Why is `\s*` needed?  Does this have something to do with `clean_lines.elided`?

Reply:
Same pattern is used elsewhere in this file. We want to handle even spaces before the #include